pkg/arvo/mar/chat/hook-update.hoon

```diff
@@ -1,13 +1,14 @@
-/+  *chat-json
-|_  upd=chat-hook-update
+/-  *chat-hook
+/+  *chat-hook
+|_  upd=update
 ++  grow
   |%
-  ++  json  (hook-update-to-json upd)
+  ++  json  (update:enjs upd)
   --
 ::
 ++  grab
   |%
-  ++  noun  chat-hook-update
+  ++  noun  update
```

Comment:
Something I don't love here is these are in fact being sent over the wire as `%chat-hook-update`, which might warrant some verbosity in at least the marks and !<s for consistency. (Alternately the mark system should perhaps have some manner of namespacing… even like, `%chat-hook--update` would make intermittent usage of the bare name less awkward)